index.html

```diff
@@ -53,23 +57,184 @@ <h2>I'm a Software Developer</h2>
+      </section>
+
+      <section class="works-item">
+        <article>
+          <h1 class="heading">Professional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups
+            required. Has been the industry's standard.
+          </p>
+          <ul class="works-item-list">
+            <li>HTML</li>
+            <li>Bootscrap</li>
+            <li>Ruby</li>
+          </ul>
+        </article>
+        <button type="button" class="works-btn btn">See project</button>
+      </section>
+      <section class="works-item">
+        <article>
+          <h1 class="heading">Professional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups
+            required. Has been the industry's standard.
+          </p>
+          <ul class="works-item-list">
+            <li>HTML</li>
+            <li>Bootscrap</li>
+            <li>Ruby</li>
+          </ul>
+        </article>
+        <button type="button" class="works-btn btn">See project</button>
+      </section>
+
+      <section class="works-item">
+        <article>
+          <h1 class="heading">Professional Art Printing Data</h1>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups
+            required. Has been the industry's standard.
+          </p>
+          <ul class="works-item-list">
+            <li>HTML</li>
+            <li>Bootscrap</li>
+            <li>Ruby</li>
+          </ul>
+        </article>
+        <button type="button" class="works-btn btn">See project</button>
+      </section>
+
+    </section>
+
+
+    <section class="about-section">
+      <article class="about-details">
+        <h1>About me</h1>
+        <p>Hello I’m a software developer! I can help you build a product,
+          feature or website Look through some of my work and experience!
+          If you like what you see and have a project you need coded, don’t
+          hesitate to contact me.
+        </p>
+        <button type="button" class="about-btn btn">Get my resume</button>
+      </article>
+      <hr>
+      <article class="about-list">
+        <div class="language">
+          <h3 class="heading">Language</h3>
+          <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+        </div>
+        <div class="framework">
+          <h3 class="heading">Frameworks</h3>
+          <ul>
+            <li>React.js</li>
+            <li>Ruby on Rails</li>
+            <li>RSpec</li>
+            <li>Capybara</li>
+          </ul>
+        </div>
+        <div class="skills">
+          <h3 class="heading">Skills</h3>
+          <ul>
+            <li>Data Management</li>
+            <li>Version Control</li>
+            <li>CLI</li>
+            <li>API Design</li>
+            <li>Web Development</li>
+          </ul>
```

Comment:
You did well with your project and your design is close to the figma template given. please for Google search optimization and accessibility reasons, it's always advisable to use one `h1` tag for your project, so kindly effect the changes for these sections(works and about) of your project highlighted ,since you already have the the `h1` tag in  your headline section😄

Reply:
@usorfaitheloho Thanks for the feedback. I have a question though. I used h3 tags like so because I used separate div tags. So can you please explain better to me what you are suggesting?

Reply:
I actually highlighted lines 75 to 193, but it didn't reflect. That's where the issue is. you used `h1` tags a lot there

Reply:
Thank you. It's clearer now

Reply:
you're welcome